src/_premake_init.lua

```diff
@@ -768,6 +768,17 @@
 		}
 	}
 
+	api.register {
+		name = "swiftversion",
+		scope = "config",
+		kind = "string",
+		allowed = {
+			"4.0",
```

Comment:
This looks like a maintenance hell, there's [a lot of versions available](https://en.wikipedia.org/wiki/Swift_(programming_language)#Version_history) :( Should we allow anything?

Reply:
There is a lot, but Xcode 11 only allows you to select those 3
I something similar to the c++11, c++14, c++17,... in the block above

Reply:
Oh, ok! The main difference I see is that a new C++ standard is added every 3 years, but a new Xcode version is added every year, and several Swift versions may appear during that time ;)
Let's keep it like this for now and we can change it when we get tired of it :P